CPP17.md

```diff
@@ -365,6 +366,17 @@ m.insert(std::move(e));
 // m == { { 1, "one" }, { 3, "three" }, { 4, "two" } }
 ```
 
+### Parallel algorithms
+Many of the STL algorithms, such as the `copy`, `find` and `sort` methods, started to support the so called "parallel execution policies": `seq`, `par` and `par_unseq` which translate to "sequentially", "parallel" and "parallel unsequenced".
```

Comment:
Please remove the `so called`. Also make `"parallel execution policies"` italicized and remove the double quotes.